wdl-analysis/src/scope.rs

```diff
@@ -221,28 +163,25 @@ pub struct Struct {
     /// This is either the name of a struct definition (local) or an import's
     /// URI or alias (imported).
     span: Span,
+    /// The offset of the CST node from the start of the document.
+    ///
+    /// This is used to adjust
```

Comment:
Used to adjust?

Reply:
Whoops, must have context switched away right in the middle of completing that sentence 😅. I'll fix!